pythonfmu/tests/pythonslave.py

```diff
@@ -0,0 +1,26 @@
+from pythonfmu.fmi2slave import Fmi2Slave, Fmi2Causality, Integer, Real, Boolean, String
+
+slave_class = "PythonSlave"  # REQUIRED - Name of the class extending Fmi2Slave
+
+
+class PythonSlave(Fmi2Slave):
+
+    Fmi2Slave.author = "John Doe"
```

Comment:
In the README `Fmi2Slave.` is omitted. Both correct?

Reply:
Actually the parent class variable should not be overridden - if two different subclasses were to be defined in the same Python context, they will get the same metadata (guid, author,...). The intent is for each subclass to get its own value.